MediaRecorder.html

```diff
@@ -442,6 +443,38 @@
               <em>Return type:</em> <code>boolean</code>
             </div>
           </dd>
+
+          <dt><dfn><code>getSupportedTypes</code></dfn>, static</dt>
+          <dd>
+            When <code>getSupportedTypes()</code> method is invoked, the UA MUST
+            queue a task, using the DOM manipulation task source, that runs the
```

Comment:
Why does it need to queue a task at the beginning? Tests to ensure that promise isn't resolved too fast would be kind of fun (really!) but I haven't seen this pattern before.